swift_scality_backend/diskfile.py

```diff
@@ -159,13 +156,13 @@ def zero_copy_send(self, wsockfd):
             if resp.status != httplib.OK:
                 raise SproxydHTTPException(
                     'Unexpected response code: %s' % resp.status,
-                    ipaddr=ipaddr, port=port, path=self.safe_path,
+                    url=sproxyd.geturl() + self._name,
```

Comment:
I was confused by the name of the variable. in fact `sproxyd` is a `ParseResult` of a full URL including the object name. So you were right. Going to fix that soon.